docs/b2b-edition/specs/api-v3/company/company.yaml

```diff
@@ -1056,7 +1072,10 @@ paths:
       tags:
         - Company
       summary: Convert from BigCommerce CustomerGroup into Company
-      description: Create company from bigCommerce customer group. This API is disabled for MSF store
+      description: |-
+        Create company from a BigCommerce customer group. 
+        * This API is turned off for the MSF store.
```

Comment:
Is the API turned off, or this _endpoint_ not available? And does the endpoint not work at all, or does it return an error?

Reply:
If the store disable the B2B MSF feature, this API is avaliable; 
If the store enable the B2B MSF feature, this API is not available, and will return an error .